R/accounts.R

```diff
@@ -468,7 +472,7 @@ resolveAccount <- function(account, server = NULL) {
 }
 
 isShinyapps <- function(server) {
-  identical(server, "shinyapps.io")
+  identical(server, "shinyapps.io") | identical(server, "rstudio.cloud")
```

Comment:
How does this and `setAccountInfo` interact with some of the other hostnames that might be in use? I'm thinking about the staging stack.

Reply:
So for our other environments we override the `rsconnect.shinyapps_url` option which changes the url. So like this for localhost:

`options(rsconnect.shinyapps_url="http://localhost:7100/v1")`